rllib/algorithms/bc/torch/bc_torch_learner.py

```diff
@@ -0,0 +1,60 @@
+import logging
+from typing import Mapping
+
+from ray.rllib.core.learner.learner import (
+    POLICY_LOSS_KEY,
+    LearnerHyperparameters,
+)
+from ray.rllib.core.learner.torch.torch_learner import TorchLearner
+from ray.rllib.core.rl_module.rl_module import ModuleID
+from ray.rllib.policy.sample_batch import SampleBatch
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.framework import try_import_torch
+from ray.rllib.utils.nested_dict import NestedDict
+from ray.rllib.utils.torch_utils import sequence_mask
+from ray.rllib.utils.typing import TensorType
+
+torch, nn = try_import_torch()
+logger = logging.getLogger(__file__)
+
+
+class BCTorchLearner(TorchLearner):
+    """Implements torch-specific BC loss logic."""
+
+    @override(TorchLearner)
+    def compute_loss_for_module(
+        self,
+        *,
+        module_id: ModuleID,
+        hps: LearnerHyperparameters,
+        batch: NestedDict,
+        fwd_out: Mapping[str, TensorType]
+    ) -> TensorType:
+        if self.module[module_id].is_stateful():
+            # In the RNN case, we expect incoming tensors to be padded to the maximum
```

Comment:
not: Can we move these comment blocks always on top of the `if` line? Same for the `else` line/comment. Thx :)